_alerts/2017-10-09-segwit2x-safety.md

```diff
@@ -0,0 +1,134 @@
+result of this, spending money from a mobile wallet during a hard fork is dangerous. You will be
+vulnerable to many attacks as your wallet is not aware that the miners are breaking the consensus rules on the longest chain.
+
+If the contentious hard fork has failed and the supermajority of miners have broken their pledge
+to support the hard fork, it is safe to continue using mobile wallets, however you should check multiple
+sources of Bitcoin news such as this website, [/r/bitcoin](https://www.reddit.com/r/Bitcoin/), and the [Bitcoin Forum](https://bitcointalk.org)
+to make sure that it's safe to continue using your particular wallet. As a general rule, you should not perform any transactions
+during and shortly after the hard fork.
+
+## After the fork
+
+Blocks may be slower shortly after the hard fork and your transactions will take longer to confirm. You
+will have to pay a higher transaction fee if your transaction is high priority and you want it confirmed
+in the next block. You should try to get any important transactions done a few days before the hard fork [date](http://bashco.github.io/2x_Countdown/).
+A lot of websites and programs will likely pop up around the time of the hard fork offering to split your Bitcoin
+so you can have access to both your BTC and to the new altcoin created by this contentious hard fork, you should
+exercise caution as many of these services are likely to be scams trying to steal your bitcoin. As the
+hard fork has very little replay protection, most transactions you perform on the Bitcoin network will also
+be valid on the hard fork network. You should keep this in mind when transacting.
+
+Some services may continue to present and name the altcoin created by this contentious hard
+fork as Bitcoin. Users should be aware that this new currency is not Bitcoin. Bitcoin can only
+be changed with the overwhelming consensus of the entire Bitcoin community of individuals,
+miners, developers and economy. Even when there is overwhelming consensus, unless in an
+emergency, a hard fork should have at least a year notice period to give enough time for users to upgrade. This
+hard fork being adopted by the signatories of this agreement achieves none of these things. It is a rushed
+and hasty upgrade which only has minority community support and has been thoroughly rejected by users and the technical community.
+As this new altcoin is not Bitcoin, Bitcoin.org will not list wallets and services that present
+this altcoin as "Bitcoin" or "BTC" to their users after the hard fork. These services have exited
+the Bitcoin system in a contentious manner and therefore are no longer useful to our visitors.
+
+The following companies and services have pledged to adopt the contentious hard fork:
+
+## Wallets
+
++ Abra (United States)
++ Bitcoin.com (St. Kitts & Nevis)
++ **BitPay (United States)**
++ BitPesa (Kenya)
++ BitOasis (United Arab Emirates)
++ **Blockchain.info (UK)**
++ **BTC.com (China)**
++ Circle (United States)
```

Comment:
I don't think circle should be listed considering that they are only supporting Bitcoin as a grandfathered-in thing. See https://support.circle.com/hc/en-us/articles/217972003-New-Ventures-of-Old-Bitcoin